app/models/proposal.rb

```diff
@@ -270,19 +264,22 @@ def create_new_observation(user, adder, reason)
     # invalidate relation cache so we reload on next access
     self.observers(true)
     # when explicitly adding an observer using the form in the Proposal page...
-    if adder
-      add_observation_comment(user, adder, reason) unless reason.blank?
+    if adder && reason
+      add_observation_comment(user, adder, reason)
```

Comment:
OH wait how I see what you are saying. Let me double check that this has the outcomes I expected. Am getting all turned around with this `Dispatcher` class